packages/zambdas/src/ehr/create-update-medication-order/index.ts

```diff
@@ -156,29 +179,80 @@ async function updateOrder(
       };
 
       const { effectiveDateTime } = orderData || {};
+      transactionRequests.push({
+        method: 'POST',
+        url: `/MedicationStatement`,
+        // effective date time for MedicationAdministration is date of creation,
+        // and effective date time for MedicationStatement is date of medication was given/taken
+        resource: createMedicationStatementResource(
+          orderResources.medicationAdministration,
+          medicationCodeableConcept,
+          {
+            effectiveDateTime,
+          }
+        ),
+      });
+    }
+  }
 
-      resultPromises.push(
-        oystehr.fhir
-          .create<MedicationStatement>(
-            // effective date time for MedicationAdministration is date of creation,
-            // and effective date time for MedicationStatement is date of medication was given/taken
-            createMedicationStatementResource(updatedMedicationResource, medicationCodeableConcept, {
-              effectiveDateTime,
-            })
-          )
-          .then((res) => console.log('successfully created MedicationStatement: ', res.id))
-      );
+  if (interactions != null && newMedicationCopy != null) {
```

Comment:
`if (something)` is more reliable than `if (something != null)`, there are several cases in this PR.